app/routers/agenda.py

```diff
@@ -39,6 +40,21 @@ def agenda(
     """Route for the agenda page, using dates range or exact amount of days."""
 
     user_id = 1  # there is no user session yet, so I use user id- 1.
+
+    get_a_week_int = 6
+    start_date_graph, end_date_graph = calc_dates_range_for_agenda(
+        0, 0, get_a_week_int)
+
+    events_this_week = agenda_events.get_events_per_dates(
+        db, user_id, start_date_graph, end_date_graph
+    )
+    events_for_graph = {
+        str(start_date_graph +
```

Comment:
I'm sure we can make these three lines bit more readable :) Please fix the line splitting

Reply:
done